root/defaults/nginx/authentik-server.conf.sample

```diff
@@ -19,7 +19,7 @@ location = /outpost.goauthentik.io/auth/nginx {
     include /config/nginx/proxy.conf;
     include /config/nginx/resolver.conf;
     set $upstream_authentik authentik-server;
-    proxy_pass http://$upstream_authentik:9000;
+    proxy_pass http://$upstream_authentik:9000/outpost.goauthentik.io/auth/nginx;
```

Comment:
is this intended with the authelia change?

Reply:
This is a change to the authentik conf so that it uses the same method that the authelia conf does for the location receiving the auth request. I don't know if it's necessary with authentik, but it should work and not have any negative impact.